src/include/os.h

```diff
@@ -66,7 +66,7 @@
 	(WT_TIMEDIFF_NS((end), (begin)) / WT_BILLION)
 
 #define	WT_TSCDIFF_NS(s, end, begin)					\
-	(__wt_tsc_to_nsec((s), (uint64_t)(end) - (uint64_t)(begin)))
+	(__wt_tsc_to_nsec((s), (end), (begin)))
```

Comment:
I don't think we need to parenthesize `s`, `end` or `begin`.

There are other, close-by time macros with similar unnecessary parenthesis.